### PR TITLE
fix(Scripts/TempleOfAhnQiraj): Fix double spawns during Cthun phase 2 and several other fixes

### PR DESCRIPTION
**Pull request by Nyeriah**
## Changes Proposed:
-  fixes tentacle animations
-  cthun spitting out while dead (area trigger)
- jump pad logic
- double spawns in p2
- combat reach of eye and tentacles

## Issues Addressed:

- Closes 

## Tests Performed:

- 
- 

## How to Test the Changes:

1.
2.
3.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by Gultask** [comment]
Teleporting looks good now, and the jump pad feels much better.
Couldn't reproduce the double spawns or the chasing pets, either 👍

**Review by acidmanifesto — APPROVED** [review]
Approving due to Testing report by Gultask https://github.com/azerothcore/azerothcore-wotlk/pull/13413#issuecomment-1279537486.

Failure is unrelated to this PR and seems to be some goodtest issue.

**Comment by acidmanifesto** [comment]
Thank you for the PR

**Comment by Kitzunu** [comment]
Why did we merge a PR with failing CI's 🤷

File: src/server/scripts/Kalimdor/TempleOfAhnQiraj/boss_cthun.cpp [hunk 1]
```diff
@@ -985,30 +981,37 @@ class at_cthun_stomach_exit : public AreaTriggerScript
...
-                player->m_Events.AddEventAtOffset([player]()
-                {
-                    player->JumpTo(0.0f, 80.0f, false);
-                }, 5s);
+                }
 
                 player->m_Events.AddEventAtOffset([player, cthun]()
                 {
-                    if (cthun)
+                    if (Creature* trigger = player->FindNearestCreature(NPC_EXIT_TRIGGER, 10.0f))
```
Comment (Kitzunu) [inline src/server/scripts/Kalimdor/TempleOfAhnQiraj/boss_cthun.cpp:996]: trigger is unused.

**Comment by acidmanifesto** [comment]
> Why did we merge a PR with failing CI's 🤷 

Was a google test failure, but when it got merged it ended up showing a different failure on the master. I already ping nyeriah in regards to it. He will get to it whenever he has timem

**Comment by Nyeriah** [comment]
@Kitzunu It was merged with a failing build because every build is failing.

**Pull request merged**